issue_template.md

```diff
@@ -0,0 +1,18 @@
+**Description:**
+<!-- Give a brief description of the issue -->
+
+**Suggested Labels:**
+<!-- Optional comma separated list of suggested labels. Non committers can’t assign labels to issues, so this will help issue creators who are not a committer to suggest possible labels-->
+
+**Affected Versions:**
+
+**OS, DB, other environment details and versions:**    
+
+**Steps to reproduce:**
```

Comment:
I think we should order this according to the priority. Shall we reorder as below:
- Description
- Steps to reproduce
- Affected Versions
- OS, DB, other environment details and versions 
- Related Issues (Optional)
- Suggested Labels (Optional)
- Suggested Assignees (Optional)